src/me/ryanhamshire/PopulationDensity/DropShipTeleporter.java

```diff
@@ -91,21 +83,17 @@ void makeEntityFallDamageImmune(LivingEntity entity)
         {
             Player player = (Player) entity;
             if(player.getGameMode() == GameMode.CREATIVE || player.getGameMode() == GameMode.SPECTATOR) return;
-            player.setGliding(false);
```

Comment:
Hmm, this is no longer a thing? Have elytras changed drastically in 1.13 that there is no such gliding ability?

Reply:
will revert this back, seemed unneccessary to me but I see now why this was there lol.